bucket/ant.json

```
@@ -9,6 +9,11 @@
     "url": "https://www.apache.org/dist/ant/binaries/apache-ant-1.10.12-bin.zip",
     "hash": "sha512:7e6fbcc3563df4bd87c883ad86a161a71da2774e0ed71a1b3aad82cbff1a7656ed9a0acb5ce40652129376dfd79f1ef74ec3369c1067d412a63062fea62ceccd",
     "extract_dir": "apache-ant-1.10.12",
+    "pre_install": [
+        "if (Test-Path -PathType Container \"$persist_dir\\lib\") {",
```

```suggestion
        "if (Test-Path \"$persist_dir\\lib\") {",
```
`-PathType Container` looks redundant. I cannot think of any case that `lib` can be a file.

---

Accepting this. The only reason I put it there was to be explicit about what we are looking for: a dir.